src/Sentry/SentryOptions.cs

```diff
@@ -326,7 +326,7 @@ public bool Debug
         /// <remarks>
         /// The <see cref="Debug"/> flag has to be switched on for this setting to take effect.
         /// </remarks>
-        public SentryLevel DiagnosticsLevel { get; set; } = SentryLevel.Debug;
+        public SentryLevel DiagnosticLevel { get; set; } = SentryLevel.Debug;
```

Comment:
M: why don't we deprecate `DiagnosticsLevel ` and forwards getters/setters to `DiagnosticLevel`? so we won't break people's code, we could remove on the next version

Reply:
It's a breaking change and a niche option (most ppl are not fiddling with this).

Since it's a major bump and it's an easy to discover fix (even looking at intellisense ppl can figure out) I rather we just do it (optimize for our maintainalibiliy in this case).

If this was a rename on `BeforeSend` for example I'd surely not do this same thing.